engine/src/main/resources/application.properties

```
@@ -67,6 +67,15 @@ mp.messaging.incoming.exportrequests.key.deserializer=org.apache.kafka.common.se
 mp.messaging.incoming.exportrequests.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 mp.messaging.incoming.exportrequests.cloud-events=false
 
+# Output queue to drawer
+mp.messaging.outgoing.todrawer.connector=smallrye-kafka
```

We could call the channel `drawer` instead of `todrawer` since that will always be a one-way communication (no return through Kafka):
```suggestion
mp.messaging.outgoing.drawer.connector=smallrye-kafka
```